src/Storage.ts

```diff
@@ -65,7 +65,7 @@ export default class Storage {
     }
   }
 
-  verifyToken(): Promise<any> {
+  async verifyToken(): Promise<any> {
```

Comment:
just wondering if this change may have any effect on other methods, I honestly don't know.

Reply:
No, that's just for code clarity to explicitly identify it as an async function. (It must be executed with `await` anyway because it returns a promise and so it does nothing if called synchronously.)

Reply:
right, makes sense 👍 diky